### ISSUE TITLE
Secure coding guidelines hosting on erlef.org

### ISSUE THREAD

**Issue by max-au** [erlef/security-wg#20]
Right now the guide is located here: https://erlef.github.io/security-wg/secure_coding_and_deployment_hardening/

It might be better to have it published directly to erlef.org

@starbelly  what do you think?

**Comment by starbelly** [comment]
Yeah, I think that would be great. Here's another alternative. Keep posting to github, but we add a passthrough such that you go to say https://erlef.org/wg/security/guide  and you get the goods, without having to hit github yourself. The only drawback there is keeping styles in sync. The only reason I haven't done a passthrough is the different css styles. 

Thoughts?

**Comment by vkatsuba** [comment]
Here is a topic about apply CSS to `iFrame` https://redstapler.co/how-to-apply-css-to-iframe/. We can try use this approach to change CSS. Also we can use https://demos.jquerymobile.com/1.4.5/popup-iframe/ for show information in modal popups.

**Comment by maennchen** [comment]
@max-au / @starbelly Is this still relevant with the updated styles and domain?

**Comment by starbelly** [comment]
@maennchen  I don't think so anymore. I think the way this is being handled atm is fine?

**Comment by maennchen** [comment]
@starbelly I agree. Let’s close this then.
